app/i18n/nl/conf.php

```diff
@@ -102,7 +102,7 @@
 			'order' => 'Sorteren op datum',
 			'search' => 'Expressie',
 			'state' => 'Status',
-			'tags' => 'Weergeven op tag',
+			'tags' => 'Weergeven op tag',	// DIRTY
```

Comment:
Maybe @Frenzie could have a quick fix there?

Reply:
You can just replace tag with label. (Outside of software _etiket_ might be better.)